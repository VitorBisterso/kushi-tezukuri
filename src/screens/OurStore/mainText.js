export default 'Postamos nossos produtos no perfil @kushi_tezukuri no instagram e fazemos nossas vendas pelo direct! É só adicionar seus produtos por aqui e mandar uma mensagem pra gente ;)'
