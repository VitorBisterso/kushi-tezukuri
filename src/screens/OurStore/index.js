import React from 'react'

import GenericScreen from '../../components/GenericScreen'

import mainText from './mainText'
import ourStoreImage from '../../assets/img/ourStore.svg'

const OurStore = () => (
  <GenericScreen title="Nossa loja" mainText={mainText} image={ourStoreImage} />
)

export default OurStore
