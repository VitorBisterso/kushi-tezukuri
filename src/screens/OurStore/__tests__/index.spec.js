import React from 'react'
import { shallow } from 'enzyme'

import OurStore from '../index'

describe('OurStore', () => {
  it('should render correctly', () => {
    const component = shallow(<OurStore />)

    expect(component).toMatchSnapshot()
  })
})
