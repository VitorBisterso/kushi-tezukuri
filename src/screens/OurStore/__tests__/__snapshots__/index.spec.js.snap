// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurStore should render correctly 1`] = `
<GenericScreen
  image=""
  mainText="Postamos nossos produtos no perfil @kushi_tezukuri no instagram e fazemos nossas vendas pelo direct! É só adicionar seus produtos por aqui e mandar uma mensagem pra gente ;)"
  title="Nossa loja"
/>
`;
